### ISSUE TITLE
Is there a way to keep refreshing an OAuth token?

### ISSUE THREAD

**Issue by riklaunim** [simplabs/ember-simple-auth#2238]
Is there a way to periodically refresh the session/oauth token in the background? I have an ember based "admin panel" and sometimes people start using it after a period of inactivity after which their token is no longer valid. They click something, requests fail due to authentication and the panel redirects to re-authenticate which can be kind of bad experience for the user (and can lead to unsaved changes, JS errors in logs/sentry etc).

**Comment by pabloascarza** [comment]
Yes, you have the  [refreshAccessTokens](https://ember-simple-auth.com/api/classes/OAuth2PasswordGrantAuthenticator.html#property_refreshAccessTokens) property, in our case we had to reimplement the private method _refreshAccessToken but it's not needed
